src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${name}`}
+    </h2>
+    <p className="Person__age">
+      {age > 0 && (
```

Comment:
Move this condition above the tag to avoid rendering an empty tag when it is false

Reply:
Check other similar cases.